resources/persist/src/lib.rs

```diff
@@ -31,14 +31,14 @@ pub struct PersistInstance {
 }
 
 impl PersistInstance {
-    pub fn save<T: Serialize>(&self, key: &str, struc: T) -> Result<(), PersistError> {
```

Comment:
Struct is a reserved keyword, so we can't use it here, see CI failure.

Reply:
addressed 🔨